vignettes/s3-vector.Rmd

```diff
@@ -944,20 +944,44 @@ Let's assume that the `percent` class is implemented in the pizza package in the
 
 ### Getting started
 
-First, the pizza package needs to include vctrs in the `Imports` section of its `DESCRIPTION` (perhaps by calling`usethis::use_package("vctrs")`. Then we make all vctrs functions available within the pizza package by including the directive `#' @import vctrs` somewhere. You can use `usethis::use_package_doc()` to create package-level documentation in `R/pizza-doc.R`, in which case that is also the logical place to put `#' @import vctrs`. Usually, it's not good practice to `@import` the entire namespace of a package, but vctrs is deliberately designed with this use case in mind.
+First, the pizza package needs to include vctrs in the `Imports` section of its `DESCRIPTION` (perhaps by calling`usethis::use_package("vctrs")`. While vctrs is under very active development, it probably makes sense to state a minimum version.
 
-Remember to call `devtools::document()` regularly, as you develop, to regenerate `NAMESPACE` and the `.Rd` files.
+```
+Imports: 
+    a_package,
+    another_package,
+    ...
+    vctrs (>= x.y.z),
+    ...
+```
```

Comment:
I feel like it's important to feature obvious, copy-paste-able snippet for important things. Versus just embedding in prose.

Reply:
But this isn't copy and pastable? I was inserting the usethis calls since you can actually run them on your package.

Reply:
OK true not copy/pastable. But realistic re: how it should look and pulled into an obvious code chunk. I do still mention the usethis commands. I think when I read docs I often skip over the prose and just read the code (and I sometimes I suffer for that, but I can't be alone).